frontend/app/components/wp-relations/wp-relations.directive.ts

```diff
@@ -70,18 +73,12 @@ export class WorkPackageRelationsController {
       });
   }
 
-  protected getRelatedWorkPackages(workPackageIds:string[]) {
-    let observablesToGetZipped = workPackageIds.map(wpId => {
+  private getRelatedWorkPackages(workPackageIds:string[]):Observable<WorkPackageResourceInterface[]> {
```

Comment:
@oliverguenther The old code was a bit weird. Maybe you can double-check my changes?

Reply:
@romanroe You need to call zip with the observables as arguments, not as an array. I've fixed this commit that should resolve the remaining integration tests.